tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -224,7 +224,7 @@ jobs:
         inputs:
           command: 'custom'
           workingDir: ${{ parameters.testWorkspace }}
-          customCommand: 'install $(Initialize.testPackageTgz)' # ${{ parameters.loggerPackage }}'
+          customCommand: 'install $(Initialize.testPackageTgz) ${{ parameters.loggerPackage }}'
```

Comment:
@markfields I had commented this during rebase to main. I doubt it but do you think this could be causing the failures with lz4? Also, does adding this looks okay?

Reply:
I don't think it would cause the lz4 thing.  Install may fail though, which was the case originally - hence I commented it out.

Reply:
Let me check this in. If install fails, I will revert this part.